functests/common.go

```diff
@@ -96,7 +100,15 @@ func GetDataValidatorJob(pvc string) *k8sbatchv1.Job {
 								},
 							},
 							Command: []string{"/bin/sh", "-c"},
+<<<<<<< HEAD
+							Args: []string{"dd if=/dev/zero of=/tmp/random.img bs=512 count=1", //This command creates new file named random.img
+								"md5VAR1=$(md5sum /tmp/random.img | awk '{ print $1 }')",  //calculates md5sum of random.img in pod and stores it in a variable
+								"cp /tmp/random.img /data/random.img",                     //copies random.img file to pvc's mountpoint
+								"md5VAR2=$(md5sum /data/random.img | awk '{ print $1 }')", //calculates md5sum of file random.img
+								"if [[ \"$md5VAR1\" != \"$md5VAR2\" ]];then exit 1; fi"},  //compares the md5sum of random.img file
+=======
 							Args:    []string{"dd if=/dev/zero of=/tmp/random.img bs=512 count=1;md5VAR1=$(md5sum /tmp/random.img | awk '{ print $1 }');cp /tmp/random.img /data/random.img;md5VAR2=$(md5sum /data/random.img | awk '{ print $1 }');if [[ \"$md5VAR1\" != \"$md5VAR2\" ]];then exit 1; fi"},
+>>>>>>> 8e9e938f57bfc8e455a85e8df90b1f4e6da2c12b
```

Comment:
Some leftover merge artifacts got commited

Reply:
yep. let me correct it.